OPHD/Things/Structures/CommandCenter.h

```diff
@@ -33,3 +34,4 @@ class CommandCenter: public FoodProduction
 		return 0;
 	}
 };
+#endif
```

Comment:
I take it this is in preparation of moving toward a data based approach? E.g., pulling the hard coding out and putting definitions into some external file?

Reply:
Yes. This should be cleaned up when tackling issue #843.